lib/Github/Api/PullRequest.php

```diff
@@ -22,16 +22,11 @@ class PullRequest extends AbstractApi
      *
      * @link https://developer.github.com/v3/pulls/#custom-media-types
      * @param string|null $bodyType
-     * @param string|null $apiVersion
      *
      * @return self
      */
-    public function configure($bodyType = null, $apiVersion = null)
```

Comment:
Even though removing this is not breaking BC, I think we should keep it. Because if we remove it and then later adds another optional parameter to this function, that WILL be BC and it is really hard to keep track of.